### DISCUSSION TITLE
I can't create beacons

### DISCUSSION THREAD

**Discussion by poperigby** [kylecorry31/Trail-Sense#757] [Q&A]
Hey, love the app. For some reason, creating a new beacon isn't working. I go navigation>compass icon>plus button>beacon, but it does nothing. Am I doing it right?

**Comment by kylecorry31** [comment]
Hi, thank you for reporting this. It appears in the last release the clicking of those buttons was accidentally changed to the label next to the button. I'll fix this and push out a new release today.

**Reply by kylecorry31** [reply]
Just created version 2.3.2 to fix this

**Reply by poperigby** [reply]
Thanks!
